app/components/compare/platformSpecific/CompareSidebar.electron.ts

```diff
@@ -1,5 +1,21 @@
 import { remote, OpenDialogSyncOptions } from 'electron'
+import fs from 'fs'
 
-export function showOpenDialogSync (opts: OpenDialogSyncOptions): string[] | undefined {
-  return remote.dialog.showOpenDialogSync(remote.getCurrentWindow(), opts)
+export function chooseFile (opts: OpenDialogSyncOptions): string[] {
```

Comment:
for stronger type safety here use a tuple return value:

```suggestion
export function chooseFile (opts: OpenDialogSyncOptions): [string, string] {
```

This'll specify that the array length _must_ be 2